### PR TITLE
Feature: LINKTA-214 implement animation for main hero

### PR DESCRIPTION
**Pull request by ChTiSh**
## Description
Added lottie player for next to play the digital assets
I need to look through more lottiefiles docs to figure the underlying problem, but right now, the flash problem is solved

Fixes # (issue) LINKTA-214

## Type of change

- [ ] 🐛 Bug fix (non-breaking change which fixes an issue)
- [X] ✨ New feature (non-breaking change which adds functionality)
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 📝 Documentation update
- [ ] Other 🗒️ **\*\***\_\_**\*\***

## How Has This Been Tested?

- [X] Self-Testing - There are bugs
- [ ] Test B

## Checklist:

- [X] I have performed a self-review of my own code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I have reviewed the [**Security Best Practices Document**](SECURITY_BEST_PRACTICES.md).
- [ ] I have followed the [**Naming Conventions Guide**](NAMING_CONVENTIONS_GUIDE.md) for my changes.
- [ ] I have ensured that my pull request title is descriptive.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
     "cypress-run": "npx cypress run"
   },
   "dependencies": {
+    "@lottiefiles/lottie-player": "^2.0.4",
```
Comment (LucieSeidler) [inline package.json]: theres a weird merge error here, suggest merging with dev now that firebase is added

**Review by LucieSeidler — APPROVED** [review]
now we need to do formatting which will be a lift but it looks great! Thanks for all the work

**Review by zhiyi15 — APPROVED** [review]

File: app/components/common/LottieSeriesPlayer.tsx
Comment (kinskiwu) [inline app/components/common/LottieSeriesPlayer.tsx:1]: This this supposed to be tsx file? It looks like a json file to me.

File: package.json
Comment (kinskiwu) [inline package.json:1]: Looks like we have `@lottiefiles/react-lottie-player` in the code but the package has not been added to the package.json.

File: app/components/common/LottieSeriesPlayer.tsx
Comment (ChTiSh) [reply app/components/common/LottieSeriesPlayer.tsx:1]: Yeah, my local merge conflicts are going crazy right now

**Pull request merged**